File: room_reservation/lib/models/room.dart
class Room {
  String id;
  final String label;
  final String image;
  final String capacity;

  Room(
      {required this.id,
      required this.label,
      required this.image,
      required this.capacity});

  factory Room.fromMap(Map<String, dynamic> map) {
    try {
      return Room(
      id: map['_id'],
      label: map['label'],
      capacity: "${map['capacity']}",
      image:map['image']
    );
    } catch (e) {
      rethrow;
    }
  }
}

File: server/src/models/room.js
const mongoose = require("mongoose");

const Schema = mongoose.Schema;
let room ="room.jpg"
const RoomSchema = new Schema({
    label: { type: String, required: true },
    capacity: { type: Number, required: true },
    image: { type: String ,default: room},
});
module.exports = mongoose.model("room", RoomSchema);
